spoken_to_signed/text_to_gloss/spacylemma.py

```diff
@@ -0,0 +1,36 @@
+from .types import Gloss
+from .common import load_spacy_model
+
+
+LANGUAGE_MODELS_SPACY = {
+    "de": "de_core_news_lg",
+    "fr": "fr_core_news_lg",
+    "it": "it_core_news_lg",
+    "en": "en_core_web_lg",
+}
+
+
+def text_to_gloss(text: str, language: str, ignore_punctuation: bool = True) -> Gloss:
```

Comment:
I vote against `ignore_punctuation=True` being the default.
For example,
```py
import spacy

nlp = spacy.load("en_core_web_sm", disable=("parser", "ner"))

doc = nlp("testing @ 1 2 3")
for token in doc:
    print(token.text, token.lemma_, token.is_punct)
```

Says that the "@" symbol is punct, but in truth I would like it to match a video like https://media.spreadthesign.com/video/mp4/13/153076.mp4 

Having the lemmatizer return everything, and then the `gloss_to_pose` section decide what to use seems better to me

Reply:
Capitalization is important, I think. Especially for German (Essen vs essen). The `gloss_to_pose` module does prefer the same capitalization, and reverts to lowercase if no exact same capitalization